docs/database-dynamodb-single-table.md

```diff
@@ -0,0 +1,92 @@
+# Database - DynamoDB Single Table
+
+The `database/dynamodb-single-table` construct deploys a single DynamoDB table with pre-configured indexes following [Single Table Design](https://www.alexdebrie.com/posts/dynamodb-single-table/) principles.
+
+## Quick start
+
+```yaml
+service: my-app
+provider:
+  name: aws
+
+constructs:
+    myTable:
+        type: database/dynamodb-single-table
+
+plugins:
+    - serverless-lift
+```
+
+On `serverless deploy`, a preconfigured DynamoDB table will be created.
+
+## How it works
+
+The `database/dynamodb-single-table` construct creates and configures the table for production:
```

Comment:
Even though it's repeating the introduction paragraph, this piece of information might be very important to help new users understand why everything is so locked down "in a weird way" (at least until you've learned what single-table design is). So that might be worth repeating:

```suggestion
The `database/dynamodb-single-table` construct creates and configures the table for production following [Single Table Design](https://www.alexdebrie.com/posts/dynamodb-single-table/) principles:
```